src/patterns/technical-problem-pages/offline-support.njk

```diff
@@ -0,0 +1,28 @@
+---
+layout: layout-example.njk
+---
+
+<div class="govuk-o-width-container">
+
+  <div class="govuk-o-main-wrapper">
+    <div class="govuk-o-grid">
+      <div class="govuk-o-grid__item govuk-o-grid__item--two-thirds">
+
+      <h1 class="govuk-heading-xl">Sorry, there is a problem with the service</h1>
+      <p class="govuk-body-l">Try again later.</p>
+      <p class="govuk-body">Contact us if you have any questions.</p>
+      <p class="govuk-body govuk-!-mb-r0">Telephone:</p>
```

Comment:
This looks great visually but we may want to use different HTML for the contact section.